pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -679,6 +690,11 @@ func (r *Builder) mapDisks(vm *model.VM, persistentVolumeClaims []*core.Persiste
 				},
 			},
 		}
+		// For multiboot VMs, if the selected boot device is the current disk,
+		// set it as the first in the boot order.
+		if bootDisk == i+1 {
+			kubevirtDisk.BootOrder = ptr.To(uint(1))
```

Comment:
isn't the boot order start from 0?

Reply:
from the CNV VM yaml its starts with one, so this marks the disk as the bootable one.